www/pages/search_data.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+permalink: /howto/search_data
+---
+<!--- IMPORTANT: This permlink is referenced from InputApp -->
+
+You can search for the content of your attribute data from Input.
+
+# QGIS project configuration
+
+To be able to search the attribute table of a layer, you need to first configure your [QGIS project](https://docs.qgis.org/3.10/en/docs/user_manual/introduction/qgis_configuration.html?highlight=properties#data-sources-properties):
+
+- Open QGIS Desktop with you QGIS Project
+- In the Menu > Project > Properties click on "Data Sources"
+![browse_enable](../images/qgis_data_sources.png)
+- Under **Searchable** column, select the layers you'd like to search
```

Comment:
Does it work like that? I though all layers are browsable by default...?

Reply:
It seems so!